### DISCUSSION TITLE
What is the timeline for GPU runners?

### DISCUSSION THREAD

**Discussion by joelberkeley** [community/community#122573] [Actions]
I am an open source developer with a free account. I am really super keen to use the GPU runners [GitHub is rolling out](https://github.blog/changelog/2023-10-31-run-your-ml-workloads-on-github-actions-with-gpu-runners/). What is the timeline for these runners for free and teams accounts? I also notice that the waitlist for GPU runners even for teams is no longer available (the link now directs to [a page](https://resources.github.com/devops/accelerate-your-cicd-with-arm-and-gpu-runners-in-github-actions/) entirely about Arm runners, even though the URL mentions GPU). Will the waitlist reopen and if so when?

**Comment by lkfortuna** [comment]
hi @joelberkeley  - The GPU runners are available for everyone on Team and Enterprise cloud plans. We no longer have a waitlist since these are now in public beta and openly available. We do not have it on our roadmap currently to offer these for free accounts due to the expensive nature of gpu accelerated VMs

**Comment by NikhilKumar7777** [comment]
GitHub has been actively developing and expanding its support for GPU runners. As of the latest updates, NVIDIA provides GPU runners for GitHub Actions, which are currently available in public beta. These runners require jobs to run in a container and set specific environment variables to function correctly​ ([GHARunners](https://docs.gha-runners.nvidia.com/runners/))​​ ([The GitHub Blog](https://github.blog/2022-09-01-github-actions-introducing-the-new-larger-github-hosted-runners-beta/))​.

Additionally, the new, larger GitHub-hosted runners are in public beta, offering more scalable and high-performance options, though specific details about the full release timeline are not provided​

**Comment by joelberkeley** [comment]
Thanks, closing as @lkfortuna has answered my question. In that case I'll get teams
